### PR TITLE
Tsify lib/handlers/formatting.js

### PR DESCRIPTION
**Pull request by matsjla**
TypeScript code for handling formatter requests

### CODE REVIEW

File: lib/formatters/base.interfaces.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export interface FormatterInfo {
     exe: string;
     styles: string[];
     type: string;
+    version: string;
```
Comment (matsjla) [inline lib/formatters/base.interfaces.ts:30]: This one was completely missing - oops!

File: lib/formatters/index.ts [hunk 2]
```diff
@@ -23,10 +23,17 @@
...
 
 export {BaseFormatter} from './base';
 export * from './_all';
 
-export const getFormatterTypeByKey = makeKeyedTypeGetter('formatter', all);
+type FormatterClass = {
+    new (opts: FormatterInfo): BaseFormatter;
+} & Keyable;
+
+export const getFormatterTypeByKey = makeKeyedTypeGetter<FormatterClass>('formatter', all);
```
Comment (matsjla) [inline lib/formatters/index.ts:39]: Not sure how elegant this is, but I had to tell TS that getFormatterTypeByKey returned something that constructs a BaseFormatter

File: static/styles/themes/dark-theme.scss [hunk 3]
```diff
@@ -492,7 +492,8 @@ textarea.form-control {
     border-color: #474747 !important;
 }
 
-.popover .arrow::after, .popover .arrow::before {
+.popover .arrow::after,
+.popover .arrow::before {
```
Comment (matsjla) [inline static/styles/themes/dark-theme.scss:496]: Prettier change

**Review by jeremy-rifkin — APPROVED** [review]
Looks good to me

**Pull request merged**